File: standalone/lemon-hlwm/perl/01-testparams.pl
#!/usr/bin/perl
# This is a modularized config for herbstluftwm tags in lemonbar

use warnings;
use strict;

use File::Basename;
use lib dirname(__FILE__);

use helper;

# ----- ----- ----- ----- ----- ----- ----- ----- ----- ----- ----- ----
# initialize

my $panel_height = 24;
my $monitor = helper::get_monitor(@ARGV);

my $lemon_parameters = helper::get_lemon_parameters(
    $monitor, $panel_height);

print $lemon_parameters."\n";

File: standalone/lemon-hlwm/python/01-testparams.py
#!/usr/bin/env python3
# This is a modularized config for herbstluftwm tags in lemonbar

import sys
import helper

# ----- ----- ----- ----- ----- ----- ----- ----- ----- ----- ----- ----
# initialize

panel_height = 24
monitor = helper.get_monitor(sys.argv)

lemon_parameters = helper.get_lemon_parameters(monitor, panel_height)
print(lemon_parameters)
